Formula/sipp.rb

```diff
@@ -22,4 +21,8 @@ def install
     system "make", "DESTDIR=#{prefix}"
     bin.install "sipp"
   end
+
+  test do
+    assert_equal "SIPp", shell_output("#{bin}/sipp -v | grep SIPp | awk '{ print $1 }'").strip
```

Comment:
It's possible Mike's suggestion with a `2>&1` tacked onto the end after `-v` will do the trick, and is quite a bit cleaner.